common/djangoapps/student/views.py

```diff
@@ -702,8 +702,16 @@ def dashboard(request):
     course_entitlement_available_sessions = {}
     for course_entitlement in course_entitlements:
         course_entitlement.update_expired_at()
-        course_entitlement_available_sessions[str(course_entitlement.uuid)] = \
-            get_course_runs_for_course(str(course_entitlement.course_uuid))
+        # Filter only the course runs that do not have an enrollment_end date set, or have one set in the future
```

Comment:
Nit: As opposed to saying what the filtering is specifically doing, comment on why it is doing it. The code makes it clear that this is the implementation but you can say: "Do not show entitlements that are no longer enrollable."